### PR TITLE
Chore/cleanup guild page

### LINKED ISSUE #893: Clean up guild page
### What would you like to be added?

Player page:
image

Current guild page:
image

- Adjust guild page column sizes to match the player page
- Get rid of sections that don't have any data: Skills, stats, gallery, quests. Implementing these would be in their own issues, let's just hide for now

### Why is this needed?

UX standardization

### PR DESCRIPTION
**Pull request by vidvidvid**
## Overview

**Disclaimer**

I used this issue as an onboarding issue for @BojanGeorgievski  or Rrash#0514

**What features/fixes does this PR include?**

Guild page has been restyled to match the user profile.
The sections from follow up ideas are commented out.

**Please provide the GitHub issue number**

Closes #893 

## Follow up Improvement Ideas

Boxes that were hidden and should be implemented next:
- [ ] skills
- [ ] stats
- [ ] quests
- [ ] gallery

## To-do

- [x] Change the default cover image for guilds (so it's not the same as for the user profile)
- [ ] Fix the minimum width for guilds (if the description is short, the boxes become narrow as well)

## Implementation

Copied the structure from user profile and slightly adjusted it.

## Assets

Screenshot 2021-12-30 at 18 32 22

### CODE REVIEW

**Comment by firattale** [comment]
LGTM, but maybe we need another PR to fix those Github actions.

**Review by luxumbra — COMMENTED** [review]
Changes on the preview site look good and the refactor looks sane to my novice eyes. :)

I did notice that no-one shows up in any of the guilds apart from MetaFam, when most of them have players in MetaGame. Is the case for most guilds on prod too.

**Comment by vidvidvid** [comment]
true, i noticed that too, but i didn't look deeper into it. any idea why? were the non-MetaFam players ever displayed as members of a guild?

**Comment by luxumbra** [comment]
> any idea why are they not appearing? 

No idea, sorry. Even the players (penguin, peth, MD) whose guild memberships show correctly on their profiles don't show in the guild profiles. Opolis & MetaFam are the only ones that show members on prod. 

> were the non-MetaFam players ever displayed as members of a guild?
I don't think so

**Comment by dan13ram** [comment]
@vidvidvid The memberships shown in the player profile is fetched by the daoHaus subgraph, whereas the guilds are in our database alone. Therefore there is no link between the "dao" memberships and "guild" memberships. 

The guild memberships is supposed to be fetched from our discord bot, but I'm not sure how far along we are in that development.

**Comment by vidvidvid** [comment]
@dan13ram alrighty then. would be nice if we somehow fixed that once we get higher outer guild involvement.

**Review by luxumbra — APPROVED** [review]
LGTM on the FE and code looks fine to my eyes.

**Comment by dan13ram** [comment]
@vidvidvid @luxumbra guild memberships from discord is being tracked at https://github.com/MetaFam/TheGame/issues/771

File: packages/web/components/Setup/SetupTimeZone.tsx [hunk 1]
```diff
@@ -46,6 +48,14 @@ export const SetupTimeZone: React.FC<SetupTimezoneProps> = ({
     onNextPress();
   };
 
+  const [isComponentMounted, setIsComponentMounted] = useState(false);
+
+  useEffect(() => setIsComponentMounted(true), []);
+
+  if (!isComponentMounted) {
```
Comment (dan13ram) [inline packages/web/components/Setup/SetupTimeZone.tsx:55]: Why do we need to do this ?

File: packages/web/components/Guild/Section/GuildPlayers.tsx [hunk 2]
```diff
@@ -50,7 +50,7 @@ export const GuildPlayers: React.FC<Props> = ({ guildId, guildname }) => {
     <ProfileSection title="Players">
       {loadingPlayers && <LoadingState />}
 
-      {guildPlayers == null ? (
+      {guildPlayers == null || guildPlayers.length === 0 ? (
```
Comment (dan13ram) [inline packages/web/components/Guild/Section/GuildPlayers.tsx:53]: This shows up when `loadingPlayers` is true.  We want to show only loading state initially and then show either "no known players yet" or the list of players.

**Review by dan13ram — CHANGES_REQUESTED** [review]
@vidvidvid Mostly looks good. Requested for a minor fix, then we can merge.

I'd like to merge this first and rebase my code in #1012 because both PRs make changes to `BOX_TYPES`

File: packages/web/components/Setup/SetupTimeZone.tsx (hunk 1 above)
Comment (vidvidvid) [reply packages/web/components/Setup/SetupTimeZone.tsx:55]: A server side rendered page was trying to access `<SelectTimeZone>` component, which is a part of the design-system, and was not found until the client loaded. The builds and deploys failed and when I introduced this change they passed. I don't know how else to solve this problem, do you perhaps have any suggestion?

File: packages/web/components/Guild/Section/GuildPlayers.tsx (hunk 2 above)
Comment (vidvidvid) [reply packages/web/components/Guild/Section/GuildPlayers.tsx:53]: ahh yeah, gotcha. will fix that.

**Comment by alalonde** [comment]
> @vidvidvid The memberships shown in the player profile is fetched by the daoHaus subgraph, whereas the guilds are in our database alone. Therefore there is no link between the "dao" memberships and "guild" memberships.

See the first bullet-point [here](https://github.com/MetaFam/TheGame/issues/771#issuecomment-1002666049). IMO we should also populate DAO memberships. There is currently a `membership_through_discord` flag on the guild table; if that is false and a DAOhaus address exists, we could populate our guild_player table that way.

> 
> The guild memberships is supposed to be fetched from our discord bot, but I'm not sure how far along we are in that development.

Yes, existing guilds first have to link their discord servers (through the /join/guild) flow for those memberships to be populated.

**Comment by dan13ram** [comment]
> See the first bullet-point [here](https://github.com/MetaFam/TheGame/issues/771#issuecomment-1002666049). IMO we should also populate DAO memberships. There is currently a `membership_through_discord` flag on the guild table; if that is false and a DAOhaus address exists, we could populate our guild_player table that way.
I like this idea. You think we can write a cron to update this every 24hrs or so ?

> Yes, existing guilds first have to link their discord servers (through the /join/guild) flow for those memberships to be populated.
 Sounds good to me!

**Review by dan13ram — APPROVED** [review]

File: packages/web/components/Setup/SetupTimeZone.tsx (hunk 1 above)
Comment (dan13ram) [reply packages/web/components/Setup/SetupTimeZone.tsx:55]: Interesting. I tried a few different things and nothing fixed that issue. :/

Comment (vidvidvid) [reply packages/web/components/Setup/SetupTimeZone.tsx:55]: Yep, it's quite peculiar 🤷

**Pull request merged**